src/debugAdapter/goDebug.ts

```diff
@@ -819,8 +819,8 @@ class GoDebugSession extends DebugSession {
 			};
 		} else if (v.kind === GoReflectKind.String) {
 			let val = v.value;
-			if (v.value && v.value.length < v.len) {
-				val += `...+${v.len - v.value.length} more`;
+			if (v.value && Buffer.byteLength(v.value) < v.len) {
```

Comment:
We can avoid calling Buffer.byteLength twice by pulling it out say like `let byteLength = Buffer.byteLength(val || '');`